### ISSUE TITLE
Snapping toolbar: drop down list box has no accessibility name and is read as being a text box

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#4477]
### Steps to reproduce

On Windows, and with a screen reader running, tab to the drop down list box in the snapping toolbar.

### Expected behavior

1. A user of a screen reader would expect a control to have a name - in general, all controls having accessibility names is very helpful, because a user of a screen reader has less context available than a sighted user.
A suitable accessibility name for the drop down list box might be "snap to"

2. The control should be read as being a combo box.

### Actual behavior

1. The drop down list box has an empty accessibility name.

2. It is read as being a text box. So most users of screen readers won't be able to work out how to change the value of this control. (An exception are those using Narrator, as by default it reads the name accessibility of the containing parent window.)

### Audacity Version

other
https://apps.microsoft.com/store/detail/winui-3-gallery/9P3JFPWWDZRC

### Operating system

Windows

**Comment by Ryan-Miller-Audacity** [comment]
Tested and confirmed implementation of accessibility variables with NVDA screenreader.

**Comment by Ryan-Miller-Audacity** [comment]
This has been merged.
